contracts/crowdfund/ERC20LaunchCrowdfund.sol

```diff
@@ -0,0 +1,532 @@
+                        }),
+                        proposalEngine: opts.proposalEngineOpts
+                    }),
+                    opts.preciousTokens,
+                    opts.preciousTokenIds,
+                    opts.rageQuitTimestamp
+                );
+        } else {
+            return
+                opts.governanceOpts.partyFactory.createPartyWithMetadata(
+                    opts.governanceOpts.partyImpl,
+                    authorities,
+                    Party.PartyOptions({
+                        name: opts.name,
+                        symbol: opts.symbol,
+                        customizationPresetId: opts.customizationPresetId,
+                        governance: PartyGovernance.GovernanceOpts({
+                            hosts: opts.governanceOpts.hosts,
+                            voteDuration: opts.governanceOpts.voteDuration,
+                            executionDelay: opts.governanceOpts.executionDelay,
+                            passThresholdBps: opts.governanceOpts.passThresholdBps,
+                            totalVotingPower: 0,
+                            feeBps: opts.governanceOpts.feeBps,
+                            feeRecipient: opts.governanceOpts.feeRecipient
+                        }),
+                        proposalEngine: opts.proposalEngineOpts
+                    }),
+                    opts.preciousTokens,
+                    opts.preciousTokenIds,
+                    opts.rageQuitTimestamp,
+                    customMetadataProvider,
+                    customMetadata
+                );
+        }
+    }
+
+    function _finalize(uint96 totalContributions_) internal override {
+        Party _party = party;
+
+        // Finalize the crowdfund.
+        delete expiry;
+
+        // Transfer funding split to recipient if applicable.
+        uint16 fundingSplitBps_ = fundingSplitBps;
+        if (fundingSplitBps_ > 0) {
+            // Assuming fundingSplitBps_ <= 1e4, this cannot overflow uint96
+            totalContributions_ -= uint96((uint256(totalContributions_) * fundingSplitBps_) / 1e4);
+        }
+
+        // Update the party's total voting power.
+        uint96 newVotingPower = _calculateContributionToVotingPower(totalContributions_);
+        _party.increaseTotalVotingPower(newVotingPower);
+
+        emit Finalized();
+
+        ERC20LaunchOptions memory _tokenOpts = tokenOpts;
+
+        // Create the ERC20 token.
+        ERC20_CREATOR.createToken(
+            address(_party),
+            _tokenOpts.name,
+            _tokenOpts.symbol,
+            TokenConfiguration({
+                totalSupply: _tokenOpts.totalSupply,
+                numTokensForDistribution: _tokenOpts.numTokensForDistribution,
+                numTokensForRecipient: _tokenOpts.numTokensForRecipient,
+                numTokensForLP: _tokenOpts.numTokensForLP
+            }),
+            _tokenOpts.recipient
+        );
+    }
```

Comment:
Should call super finalize rather than copying everything over

Reply:
It can't because `_finalize()` includes a line to transfer all ETH to the Party which is not what we want to do.